[DOCS] Adding metadata to articles (#18153)
openvinotoolkit/openvino: docs/gapi/gapi_face_analytics_pipeline.md

@sphinxdirective

.. meta::

>    :description: Learn how to integrate a deep learning inference in a G-API 
>                  graph that you will run on a video stream to obtain data.

Reviewer: ```suggestion
   :description: Learn how to integrate a deep learning inference in a G-API 
                 graph that can be run on a video stream to obtain data.
```
